### ISSUE TITLE
A11y_PowershellEditorServices_Terminal_ScreenReader : NVDA does not read any information when we type any command in the terminal multi line editable box & press enter.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#62025]
Environment Details:
VSCode Version : 1.28.2
OS Version : Win10

Additional information : NVDA ,MAS1.3.1

Repro Steps:
1.Launch Visual Studio code desktop application.
2.Navigate to "View" item of menu bar and select "Integrated Terminal"->Navigate to "Terminal" tab and select "PowerShell Integrated Console" from combo box.
3.Navigate "Terminal panel" & press enter 
4.Start NVDA & type information in the terminal multi line editable box.

Actual:
NVDA does not read any information when we type any command in the terminal multi line editable box & press enter. If I got the error message after pressing enter NVDA does not read any information .

Expected:
NVDA should read the  information when we type any information in the terminal multi line editable box & press enter.

Recommendations:
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

User Impact:
User will not able to know the error message and whatever command will execute user will not able to get it.

MAS Reference:
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}
Does this issue occur when all extensions are disabled?: Yes

Attachment:
[A11y_PowershellEditorServices_Installation_Screenreader11111.pptx](https://github.com/Microsoft/vscode/files/2524776/A11y_PowershellEditorServices_Installation_Screenreader11111.pptx)

**Comment by Tyriar** [comment]
Duplicate https://github.com/Microsoft/vscode/issues/59794
